utils/issue-format-bot/lib/issueedit.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2017 AJ Jordan
+//
+// This file is part of the HTTPS Everywhere issue management bot.
+//
+// The HTTPS Everywhere issue management bot is free software: you can
+// redistribute it and/or modify it under the terms of the GNU Affero
+// General Public License as published by the Free Software
+// Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// The HTTPS Everywhere issue management bot is distributed in the
+// hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+// the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+// PURPOSE.  See the GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with the HTTPS Everywhere issue management bot.  If
+// not, see <http://www.gnu.org/licenses/>.
+
+'use strict';
+
+const parse = require('./parse'),
+      validate = require('./validate'),
+      labeler = require('./labeler'),
+      _ = require('lodash');
+
+// TODO make this share more with newissue.js?
+
+module.exports = function(robot, alexa) {
```

Comment:
Since you're only using `robot` to log, maybe you can just call this `log` and pass in `robot.log` in https://github.com/EFForg/https-everywhere/pull/11615/files#diff-576935d3a859d98275a1dc467716638fR34 and https://github.com/EFForg/https-everywhere/pull/11615/files#diff-576935d3a859d98275a1dc467716638fR35